src/DotNet.Status.Web.Tests/TimeLineIssueServiceLogicTests.cs

```diff
@@ -0,0 +1,212 @@
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace DotNet.Status.Web.Tests
+{
+    [TestFixture]
+    public class TimeLineIssueServiceLogicTests
+    {
+        private TimelineIssueTriageService.TimelineIssueTriageLogic _logic => new TimelineIssueTriageService.TimelineIssueTriageLogic();
+
+        [TestCase("")]
+        [TestCase("[]")]
+        [TestCase("[Movies=aha]")]
+        [TestCase("[Movies](http://that.movie.yo)")]
+        [TestCase("Build\nIndex\nRecord\n")]
+        public void ParseZeroBuildInfoFromNotTriageIssue(string body)
+        {
+            var result = _logic.GetTriageItemsProperties(body);
+            result.Should().BeEmpty();
+        }
+
+        [TestCase("[]")]
```

Comment:
Some of the test case are the same as in the test method above. This would suggest that we are testing the same behavior twice.